core/templating/template_helpers.go

```diff
@@ -398,6 +496,56 @@ func (t templateHelpers) csvCountRows(dataSourceName string) string {
 	return fmt.Sprintf("%d", numRows)
 }
 
+func (t templateHelpers) csvSQL(queryString string) []RowMap {
+	templateDataSources := t.TemplateDataSource.DataSources
+	// Parse the query string to get the SelectQuery
+	query, err := parseQuery(strings.TrimSpace(queryString), templateDataSources)
+	if err != nil {
+		// Debugging: Print the parsed query
+		fmt.Printf("Error parsing query:: %+v\n", err)
+		log.Debug("Error parsing query:", err)
```

Comment:
we should just do `log.Error` here, using the proper logging framework is more consistency and provide the full logging control and context.

Reply:
Yes, I will clean up

Reply:
Done